app/src/main/java/com/cupcake/chickenmasala/utill/Extention.kt

```diff
@@ -16,4 +16,15 @@ fun ImageView.setImage(url: String) {
 
 }
 
-
+fun Int.toFormatTime(): String{
+    val hours = this/60
+    val min = this % 60
+    var formatTime = ""
+    if (hours != 0){
+        formatTime = "${hours}h "
```

Comment:
when use h/m can't translated don't use hardcode, we need set app is response with Arabic .